src/components/screens/on-call-calendar-screen.tsx

```diff
@@ -48,7 +48,7 @@ const OnCallCalendarScreen = () => {
   const [selectedDate, setSelectedDate] = useState<DateTime>(DateTime.now());
   const [onCallPerson, setOnCallPerson] = useState<string | null>(null);
   const [selectedOnCallWeek, setSelectedOnCallWeek] = useState<OnCallWeek>();
-  const isAccountant = UserRoleUtils.isAccountant();
+  const {isAccountant} = useUserRole();
```

Comment:
Quite a few examples of lack of space for curly braces in these changes, would be nice to fix before merge